### PR TITLE
Content-manager customisation

### PR DESCRIPTION
**Pull request by soupette**
# Allow to edit Content Manager's list view from the admin

My PR is a:

💅 Enhancement
🚀 New feature

Main update on the:
- Admin
- Plugin

To follow my habit of creating huge PR, this one allows the user to change the display of the list view, enable or disable search, bulk actions, filters depending on the content-type, basically the general settings of the page. These settings are stored in the database (`core_store`).

This PR also removes the `WYSIWYG` and `InputJSON` components from the `strapi-helper-plugin` since they are really specific to the `content-manager` (for the moment) so there was no need to build them in the other plugins that use the `InputsIndex` component.

#1149

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
As always, massive PR :stuck_out_tongue: <3

File: packages/strapi-plugin-content-manager/admin/src/assets/images/icon-edit-blue.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-manager/admin/src/assets/images/icon-edit-blue.svg]: Compress this file as well

File: packages/strapi-plugin-content-manager/admin/src/translations/fr.json [hunk 2]
```diff
@@ -71,7 +84,19 @@
   "error.validation.minSupMax": "Ne peut pas être plus grand",
   "error.validation.json": "Le format JSON n'est pas respecté",
 
+  "form.Input.label": "Label",
+  "form.Input.label.inputDescription": "Cette valeur modifie celle du champs de la table",
+  "form.Input.search": "Autoriser la search",
+  "form.Input.search.field": "Autoriser la search sur ce champs",
+  "form.Input.filters": "Autoriser les filtres",
+  "form.Input.sort.field": "Autoriser le trie sur ce champs",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-manager/admin/src/translations/fr.json]: `le tri`

File: packages/strapi-plugin-content-manager/admin/src/translations/fr.json [hunk 3]
```diff
@@ -71,7 +84,19 @@
...
+  "form.Input.bulkActions": "Autoriser les bulk actions",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-manager/admin/src/translations/fr.json]: `les actions groupées`

File: packages/strapi-plugin-content-manager/admin/src/translations/fr.json [hunk 4]
```diff
@@ -71,7 +84,19 @@
...
+  "form.Input.pageEntries": "Nombre d'entrées par page",
+  "form.Input.pageEntries.inputDescription": "Note: Vous pouvez les modifier ces valeurs par modèle",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-manager/admin/src/translations/fr.json]: `Vous pouvez modifier ces valeurs par modèle` remove the word `les`

File: packages/strapi-plugin-content-manager/admin/src/translations/fr.json [hunk 5]
```diff
@@ -71,7 +84,19 @@
...
+  "form.Input.defaultSort": "Attribut de trie par défault",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-manager/admin/src/translations/fr.json]: `tri`

File: packages/strapi-plugin-content-manager/admin/src/translations/fr.json [hunk 6]
```diff
@@ -71,7 +84,19 @@
...
+
   "notification.error.relationship.fetch": "Une erreur est survenue en récupérant les relations.",
+  "notification.info.SettingPage.disableSort": "Vous devez avoir au moins un attribut de trie par défaut",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-manager/admin/src/translations/fr.json]: `tri`

**Comment by soupette** [comment]
Thanks @Aurelsicoko, I appreciate !

**Comment by derrickmehaffy** [comment]
And now I'm super excited for the new release @soupette dang you *shakes happy fist* :smile:

**Comment by redni** [comment]
I was looking forward to it, and I thank you for the next functionalities! :D However, I have a small suggestion.

So in the future, can you add language changes to all current files in the catalog instead of just English and French? Because now I really need to compare a file of English translations with a Polish translation file to look for new ones and add new lines to the file and then translate. And if changes (English) were introduced to each language file, it would be much less work and search for changes.

**Comment by lauriejim** [comment]
Hi @redni, that is a great suggestion. Will try to work on this improvement soon a possible. It's a really pain, I m agree with you.

**Comment by soupette** [comment]
@redni noted!

**Pull request merged**